simulation/src/main/java/com/palantir/dialogue/core/Simulation.java

```diff
@@ -80,6 +81,10 @@ public Ticker clock() {
         return ticker; // read only!
     }
 
+    public ScheduledExecutorService scheduler() {
+        return deterministicExecutor;
+    }
```

Comment:
Hang on exposing this directly might interact weirdly with the simulator, because the above `schedule` methods ensure that the clock is set correctly, whereas direct access means that a scheduled runnable could call clock.read() and get an old value!!

Reply:
ack